spec/src/main/asciidoc/architecture.adoc

```diff
@@ -97,6 +97,23 @@ Tags have taken over the role to, for example, identify an application (`app=myS
 (`tier=database` or `tier=app_server`) and also the node/container id. Aggregation of metrics can then work over label
 queries (Give me the API hit count for `app=myShop && tier=app_server`).
 
+In MicroProfile Metrics, tags add an additional dimension to metrics that share a common basis. For example, a metric named
+`carCount` can be further differentiated by the car type (sedan, SUV, coupe, and etc) and the colour (red, blue, white, black,
+and etc). Rather than incorporating this in the metric name, tags can be used to capture these information in separate metrics.
+
+[source]
+```
+carCount{type=sedan,colour=red}
+carCount{type=sedan,colour=blue}
+carCount{type=suv,colour=red}
+carCount{type=coupe,colour=blue}
+```
+
+The key name for the tag must match the regex `[a-zA-Z_][a-zA-Z0-9_]*` (Ascii alphabet, numbers and underscore). If an illegal character is
```

Comment:
What would be the implications of UTF-8 instead of Ascii? I guess Prometheus would not allow that?